components/RadioGroup/RadioButton.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ButtonGroupLabel from '../shared/ButtonGroupLabel';
+import HiddenRadio from '../shared/HiddenRadio';
+import Icon from '../Icon';
+import uniqId from '../shared/uniqId';
+
+const LockIcon = styled(Icon).attrs({
+  name: 'lock',
+})``;
+
+const ButtonIcon = styled(Icon)`
+  margin-right: 8px;
+`;
+
+const Wrapper = styled.div`
+  ${({ inline }) =>
+    inline &&
+    `
+    display: inline-block;
+    vertical-align: top;
+    margin-right: 16px;
+
+    :last-child {
+      margin-right: 0;
+    }
+  `}
+
+  ${LockIcon} {
+    margin-left: 8px;
+    font-size: 17px;
+  }
+`;
+
+const Radio = ({
+  children,
+  label,
+  error,
+  disabled,
+  onChange,
+  value,
+  checked,
+  icon,
+  id,
+  inline,
+  ...rest
+}) => {
+  const skin = checked ? 'primary' : 'secondary';
+  const _id = id || uniqId('radio-button-');
+
+  return (
+    <Wrapper inline={inline}>
+      <HiddenRadio
+        checked={checked}
+        disabled={disabled}
+        id={_id}
+        onChange={e => onChange({ value, label }, e)}
+        value={value}
+        skin={skin}
+        error={error}
+        {...rest}
+      />
+      <ButtonGroupLabel
```

Comment:
Is too much insert a `role="button"` on this label?
Because in this scenario, its plays the role of a button.
Do you agree?

Reply:
I don't know if I agree, because we only apply button skin here. The element still a radio input. WDYT?

Reply:
i dont know to say...
That`s a issue.
Anyway, lets keep like this.